src/ragas/testset/synthesizers/generate.py

```diff
@@ -42,24 +44,32 @@ class TestsetGenerator:
     ----------
     llm : BaseRagasLLM
         The language model to use for the generation process.
+    embedding_model: BaseRagasEmbeddings
+        Embedding model for generation process.
     knowledge_graph : KnowledgeGraph, default empty
         The knowledge graph to use for the generation process.
     """
 
     llm: BaseRagasLLM
+    embedding_model: BaseRagasEmbeddings
```

Comment:
Ok thanks for the quick review. The reason I added `embedding_model` as an attribute is because in line 61, I made it so you can pass your embedding model to the `from_langchain` method which returns an instance of TestsetGenerator, containing both the LLM and embedding model (line 68). It allows the TestsetGenerator object to have an embedding model attributed to it, so if the user does not pass an embedding model to the `generate_with_langchain_docs` it can use the embedding model that is attributed to the class, and the method will not throw an error. 

Further explanation below.

Reply:
I think it is okay for now - merging this in 🙂